marlowe/doc/playground-overview.adoc

```diff
@@ -19,119 +19,172 @@ contract in the browser-based tool, the Marlowe Playground, a web tool
+
+* *Open Project* This opens a project that has been saved previously. See the section on <<_saving_and_opening_projects>> below for more details on setting this up.
+* *Open Example* This will load an example into the existing project, in the environment chosen by the user.
+
+image:open-example.png[Opening an example,width=600,align="center"]
+
+* *Rename Project* Renames a project: the name is shown in the centre of the title bar.
+* *Save Project* Saves a project. See the section <<_saving_and_opening_projects>> below.
+* *Save as New Project* Saves the current project with a new name.
+
+The program editor used in the Playground is the Monaco editor https://microsoft.github.io/monaco-editor/ and many of its features are available, including the menu available on right-click.
+
 === The Haskell Editor: developing embedded contracts
 
-The editor supports the development of Marlowe contracts, described in Haskell. We can use Haskell to make contract definitions more readable by using Haskell definitions for sub-components, abbreviations, and simple template functions. The Haskell editor tab is shown in the following image.
+The editor supports the development of Marlowe contracts described in Haskell. We can use Haskell to make contract definitions more readable by using Haskell definitions for sub-components, abbreviations, and simple template functions. The Haskell editor is shown in the following image.
 
 image:haskell-editor.png[The Haskell editor]
 
-The figure shows that the tool is in the Haskell editor pane (number 1 in the diagram). The tool contains a number of example contracts, and in this case we are looking at the “escrow” contract (number 2). To describe a Marlowe contract in the editor, we have to define a top-level value `contract` of type `Contract` (number 3); it is this value that is converted to pure Marlowe with the *Compile* button (number 4), and which we can *Send to simulator* with the eponymous button (number 5).
+The Haskell editor is open here on the Escrow example contained in the examples. To describe a Marlowe contract in the editor, we have to define a top-level value `contract` of type `Contract`; it is this value that is converted to pure Marlowe with the *Compile* button (in the top RH corner). If compilation is successful, the compiled code is shown in an overlay (which can be minimised):
 
-=== Simulating Marlowe contracts
+image:haskell-compiled.png[Haskell code compiled to Marlowe]
 
-The second tab in the Marlowe Playground allows users to simulate
-Marlowe contracts transaction by transaction. Typically these are
-contracts developed in the Haskell editor or within Blockly, but they can also be entered and modified in the editor pane here too.
+On successful compilation the result can be sent to the simulator or to Blockly: these options are provided by the *Send to Simulator* and *Send to Blockly* buttons in the top RH corner of the page.
```

Comment:
I wouldn't abbreviate RH in this context

Reply:
replaced throughout. thanks